sources/packages/web/src/views/aest/institution/ApplicationOfferingChangeRequestForm.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <full-page-container layout-template="centered-card-tab">
```

Comment:
Just wondering shouldn't it be `LayoutTemplates.CenteredTab`?

Reply:
I think it would be `LayoutTemplates.CenteredCardTab` because each of the two tabs need to be inside `v-cards`.